tests/test_main.py

```diff
@@ -95,7 +99,10 @@ def test_add_fail():
 
 
 def copy_NC_036113(url, path):
```

Comment:
This function should have a different name now.

Reply:
It has been done now.